compose/ui/ui-graphics/src/skikoMain/kotlin/androidx/compose/ui/graphics/layer/SkiaGraphicsLayer.skiko.kt

```diff
@@ -245,17 +271,19 @@ actual class GraphicsLayer internal constructor() {
     }
 
     internal actual fun draw(canvas: Canvas, parentLayer: GraphicsLayer?) {
-        if (isReleased) {
-            return
-        }
+        if (isReleased) return
 
+        var restoreCount = 0
         parentLayer?.addSubLayer(this)
 
         picture?.let {
             configureOutline()
 
             updateMatrix()
+
             canvas.save()
+            restoreCount++
+
             canvas.concat(matrix)
             canvas.translate(topLeft.x.toFloat(), topLeft.y.toFloat())
 
```

Comment:
In `RenderNodeLayer`, shadow and clipping are part of the picture. Can we move them also in this class to the picture?

Reply:
https://youtrack.jetbrains.com/issue/CMP-6618/GraphicLayer.-Consider-caching-shadow-and-clipping-in-picture-during-record-call